### ISSUE TITLE
ViewChild decorator doesn't work anymore due to EmbeddableWidget decorator

### ISSUE THREAD

**Issue by KyorCode** [digipolisantwerp/embeddable-widgets_component_angular#1]
```javascript
@Component({
  templateUrl: './form-renderer.page.html'
})
@EmbeddableWidget('definitions/form-renderer.json')
export class FormRendererPage implements AfterViewInit {
  schema: FormSpec;
  form: FrForm;
  step = 1;
  autoStep = true;

  private props = null;

  @ViewChild(FrFormComponent)
  public frFormComponent: FrFormComponent;

  public ngAfterViewInit(): void {
    debugger;
  }

...
```

The frFormComponent will never be defined due to the way the constructor is called in the decorator.

**Comment by jsebrech** [comment]
Please include form.renderer.page.html here, because I cannot reproduce this locally.

**Comment by jsebrech** [comment]
Issue only occurs in non-AOT build

**Comment by jsebrech** [comment]
Fixed in 1.0.1
